static/sass/_pattern_application.scss

```diff
@@ -87,8 +87,9 @@
         }
       }
     }
-    
-    .p-accordion__tab {
+    // TODO : the accordion component needs to be fixed in vanilla. The issue was filed on https://github.com/canonical/vanilla-framework/issues/4563. This styles written locally needs to be removed after vanilla update.
+    .p-accordion__tab span {
```

Comment:
I would put a class like u-default-max-width. We shouldn't use element selectors.

Reply:
Thanks @lyubomir-popov. Fixed!